src/main/java/com/github/jnidzwetzki/bitfinex/v2/entity/currency/BitfinexCurrencyPair.java

```diff
@@ -177,9 +177,17 @@ private static String buildCacheKey(final String currency1, final String currenc
 	 */
 	private double minimumOrderSize;
 
+	/**
+	 *	If the currencypair is one of the derivatives
+	 */
+	private boolean isFuture = false;
```

Comment:
According to the Java name scheme please not use `Is` in field names. `Is` should only be used in the getter.

Variable: `future`
Method: `public boolean isFuture`